### ISSUE TITLE
Form won't submit

### ISSUE THREAD

**Issue by srtalbot** [cds-snc/platform-forms-client#218]

**Comment by srtalbot** [comment]
To replicate:

- Mac OS, Chrome, staging environment
- Public Health Agency of Canada Cloud Services Request form: https://forms-staging.cdssandbox.xyz/id/48
- Complete all required fields, submit

Error message:
![image](https://user-images.githubusercontent.com/7973770/121965963-daa5de00-cd3b-11eb-98e0-db5ac5119047.png)

**Comment by bryan-robitaille** [comment]
Once the prototyped changes in Sandbox are merged into Staging this error should disappear.  AWS blocks any request body that is larger then 10,240 Bytes.  The majority of the current request is made up of the `formInfo` which includes the template information.  The new Template API remove this information from the request.

**Comment by bryan-robitaille** [comment]
@srtalbot Could you please test again?  With the changes from Sandbox now in Staging this should be hopefully solved.  If it now works we can close this issue.

**Comment by srtalbot** [comment]
Couldn't find the same form as the ID numbers migrated. I tried all the the numbers in sequence and still couldn't find it. 

To substitute I tried a G&C form which went through. My only question is if the additional required fields would increase the request body size such that it would be blocked on AWS. (The form example above has a lot more required questions than the G&C one)

**Comment by bryan-robitaille** [comment]
Tested In Staging - form now submits and is processed by reliability queue
